config/account.go

```diff
@@ -295,5 +300,47 @@ func (a *AccountChannel) IsSet() bool {
 }
 
 type AccountPrivacy struct {
-	AllowActivities AllowActivities `mapstructure:"allowactivities" json:"allowactivities"`
+	AllowActivities *AllowActivities `mapstructure:"allowactivities" json:"allowactivities"`
+	IPMasking       IPMasking        `mapstructure:"ip_masking" json:"ip_masking"`
+}
+
+type IPMasking struct {
+	IpV6 IPMasks `mapstructure:"ipv6" json:"ipv6"`
+	IpV4 IPMasks `mapstructure:"ipv4" json:"ipv4"`
+}
+
+type IPMasks struct {
+	ActivityLeftMaskBits    int `mapstructure:"activity_left_mask_bits" json:"activity_left_mask_bits"`
+	GdprLeftMaskBitsLowest  int `mapstructure:"gdpr_left_mask_bits_lowest" json:"gdpr_left_mask_bits_lowest"`
+	GdprLeftMaskBitsHighest int `mapstructure:"gdpr_left_mask_bits_highest" json:"gdpr_left_mask_bits_highest"`
+}
```

Comment:
Comparing to PBS-Java, they don't have `activity_left_mask_bits` and instead have just `always-mask-right` and `anon-left-mask-bits`. Why aren't we matching that?

Reply:
Yes, this is something that we have different.
`always-mask-right` (default value is 64):
They always cut 64 right bits, which is a device MAC address or device identifier.
(For visualization) it turns 
`1111:2222:3333:4444:5555:6666:7777:8888`
to 
`1111:2222:3333:4444::`

`anon-left-mask-bits` (default value is 56)
Is used in activity IPv6 scrubbing. 
It turns 
`1111:2222:3333:4444:5555:6666:7777:8888`
to 
`1111:2222:3333:4400::`

In PBS we have IPv6 configs for activities and 2 more configs for GDPR scrubbing strategies: 
```go
// ScrubStrategyIPV6Lowest16 zeroes out the last 16 bits of an IPV6 address.
ScrubStrategyIPV6Lowest16

// ScrubStrategyIPV6Lowest32 zeroes out the last 32 bits of an IPV6 address.
ScrubStrategyIPV6Lowest32
```
This is why I added `gdpr_left_mask_bits_lowest` and `gdpr_left_mask_bits_highest`.  I didn't find corresponding code in PBS-Java.

I also think lowest 16 or lowest 32 should be scrubbed from network address, not from device identifier. I might be wrong about this.

Current PBS `scrubIPV6Lowest16Bits` turns 
`1111:2222:3333:4444:5555:6666:7777:8888`
to 
`1111:2222:3333:4444:5555:6666:7777:0`

current PBS `scrubIPV6Lowest32Bits` turns 
`1111:2222:3333:4444:5555:6666:7777:8888`
to 
`1111:2222:3333:4444:5555:6666:0:0`

Instead it should be 
`scrubIPV6Lowest16Bits`: `1111:2222:3333::`
`scrubIPV6Lowest32Bits`: `1111:2222::`

These both values are easy to change in configs now. We just need to figure out what it should be.

Reply:
Here is a possible config change:
```go
v.SetDefault("account_defaults.privacy.ip_masking.ipv6.activity_left_mask_bits", 56)
v.SetDefault("account_defaults.privacy.ip_masking.ipv6.gdpr_left_mask_bits_lowest", 48) // now 112 (112-64=48)
v.SetDefault("account_defaults.privacy.ip_masking.ipv6.gdpr_left_mask_bits_highest", 32) // now 96 (96-64=32)
```

Reply:
Discussed offline. Only `left_mask_bits` variable should be present in ip masking config.

Reply:
Why are not using the same  `always-mask-right` and `anon-left-mask-bits` approach that PBS-Java has? I like the names you've been using better than though, since "mask-right" doesn't look like it ever makes actually masks the right side?

Reply:
In Java there 2 variables used in IPv6 processing: 
```yaml
ipv6:
  always-mask-right: 64
  anon-left-mask-bits: 56
```
If I understand the Java code correctly, they always remove 64 bits from the right for any incoming IPv6 address, even if GDPR is not applied:
`1111:2222:3333:4444:5555:6666:7777:8888` -> `1111:2222:3333:4444::`

If GDPR is applied and IPv6 should be anonymized only 56 bits from the left will remain. This is what we have in this PR. We only modify IPv6 if GDPR or activity is enforced: 
`1111:2222:3333:4444:5555:6666:7777:8888` -> `1111:2222:3333:4400::`

Do we also want to remove 64 bits from the right even if GDPR was not applied?